packages/edit-site/src/components/resizable-frame/index.js

```diff
@@ -82,6 +82,7 @@ function ResizableFrame( {
 	isOversized,
 	setIsOversized,
 	isReady,
+	hasResize = true,
```

Comment:
That's a weird prop in ResizableFrame component. Do we need to use it if we don't want to resize?

Reply:
Yeah, I couldn't work it out either. I'm assuming we don't want this quite yet! 😄 


https://github.com/user-attachments/assets/0d537490-bf86-461c-9245-6c174fda8daf